src/main/webapp/popular.html

```diff
@@ -3,15 +3,23 @@
 <head>
   <title>Popular Videos</title>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
-  <div id="content">
-    <label for="region">Choose a region:</label>
-    <select name="region" id="region-select">
-      <option value="US">United States</option>
-    </select>
-    <div id="popular-list-container"></div>
+<body onload="createMenu(); createHeader()">
```

Comment:
Nit: Having more than one method in html seems smelly to me. I'm conflicted between keeping static html in this html instead of dynamically creating in js. It seems like these two methods create static content. I would ask Amir for an opinion on this.

Reply:
now calling one function (addMenuHeader()) that calls function to create a menu and header.
I made it dynamic because as I was editing and making changes I had to apply them to every html page, which was a pain. Curious to get @FuzzyTree 's opinion on this though, since I know this method requires that js be loaded in order for the user to navigate the website which is a downside of dynamically creating these elements